validation_protocol.py

```
@@ -260,8 +260,8 @@ def _validate_long_accuracy(self, feature_name: str, test_results: Dict[str, Tes
         avg_long_accuracy = sum(long_accuracies) / len(long_accuracies)
         min_long_accuracy = min(long_accuracies)
 
-        # LONG accuracy трябва да е >= 95%
-        passed = min_long_accuracy >= 95.0
+        # LONG accuracy трябва да е >= 100% (перфектна точност)
```

Ruff (0.12.2)

Comment contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF003)